### PR TITLE
Rf07 tc 035

### PR DESCRIPTION
**Pull request by ivanMise**

### CODE REVIEW

**Comment by ivanMise** [comment]
image

image

File: src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java [hunk 1]
```diff
@@ -50,4 +54,20 @@ public void seleccionarSubMenuUsuario() throws Throwable {
...
+    }
+
+    @And("^Seleccionar submenu 'Equipamiento' en menu 'Equipamiento'$")
+    public void seleccionarSubMenuEquipamiento() throws Throwable {
+        listEquipment = menuEquipamiento.selectSubMenuEquipment();
+    }
+
+    @Then("^Validar que la 'Lista de Equipamientos' este visible$")
+    public void validarListaDeEquipamientos() throws Throwable {
+        boolean result = listEquipment.isEquipmentListVisible();
```
Comment (marcelogaray) [inline src/test/java/org/umssdiplo/automationv01/stepdefinitionproject/StepsDefinitionSSID.java]: Assert.assertTrue

**Review by marcelogaray — CHANGES_REQUESTED** [review]
review observation

**Comment by ivanMise** [comment]
Ya solucione las observaciones Ing. 

image

**Review by marcelogaray — APPROVED** [review]

**Pull request merged**